canonical-web-and-design/ubuntu.com
# Fix contact us title and redirects for smartstart

## Done

- Fix contact us title for /core/smartstart/contact-us as it had a single quote but was delimited by single quotes
- Fixed the redirects for /smartstart and /smart-start that missed the contact-us and thank-you page redirects

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8001/core/smart-start/contact-us
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- See that the h1 is visible


## Issue / Card

Fixes #9187
